state/indexer/block/indexer.go

```diff
@@ -20,28 +20,28 @@ import (
 // configuration.
 //
 //nolint:lll
-func IndexerFromConfig(cfg *config.Config, dbProvider config.DBProvider, chainID string) (txindex.TxIndexer, indexer.BlockIndexer, error) {
+func IndexerFromConfig(cfg *config.Config, dbProvider config.DBProvider, chainID string) (txindex.TxIndexer, indexer.BlockIndexer, dbm.DB, error) {
```

Comment:
We'll need a specific changelog entry for this, given that it's a Go API-breaking change.